test/k8sT/Policies.go

```diff
@@ -391,6 +399,144 @@ var _ = SkipDescribeIf(func() bool {
 				helpers.CurlFail("http://%s/private", clusterIP))
 			res.ExpectFail("Unexpected connection from %q to 'http://%s/private'",
 				appPods[helpers.App3], clusterIP)
+
+			By("Testing ICMP rules")
```

Comment:
Note I don't think the policies loaded by previous tests were all removed. Maybe we should fix that in a separate commit? Otherwise the ICMPs could be allowed by some L3 rule.

Reply:
I agree, but what does "in a separate commit" mean? Don't we fix and update the existing commit?
Also, do you think we should fix it by using another `It()` for ICMP testing?

Reply:
> I agree, but what does "in a separate commit" mean?

IMO it's not really an issue introduced by your code, so it could be in a separate commit.

>  Also, do you think we should fix it by using another `It()` for ICMP testing?

Another `It` block would also work. It has the advantage of clearly indicating what failed when it failed :-)

Reply:
I see, so now I'll divide it by another `It`.
And if an issue occur, let's fix it in a separate commit.

Reply:
Removing the policies at the end of the test is already happening from the 
```go
		AfterEach(func() {
			cmd := fmt.Sprintf("%s delete --all cnp,ccnp,netpol -n %s", helpers.KubectlCmd, namespaceForTest)
			_ = kubectl.Exec(cmd)
		})
```
so I don't think an additional commit is necessary.

Reply:
Neither do I. Now I divided ICMP tests and others by `It` blocks, so probably we don't have to worried about that since the `AfterEach` was executed before ICMP `It` block starts.